server/models/views/location.js

```diff
@@ -44,48 +45,83 @@ class ViewModel {
       })
 
       // Get the most severe severity (not highest since FWIS api change) as the groups are order by most severe
-      const mostSevere = groups[0].severity.id
+      // const mostSevere = groups[0].severity.id
```

Comment:
There are quite a few commented out lines of code that need to be removed on this file

Reply:
Redundant and commented out code removed.